infrastructure/api/exceptions.py

```diff
@@ -0,0 +1,11 @@
+from flask import request, jsonify
+
+
+class ApiException(Exception):
+    def __init__(self, message, code):
+        self.message = message
+        self.code = code
+
+    def handle(self):
+        request.logger.error(self.message, code=self.code, exception=self.__class__.__name__)
```

Comment:
`request` has the `logger` attribute because `SwipeFoodAPI` inserts it for every request